### DISCUSSION TITLE
[BUG] Changing BGM menu music through Tadpole appears "reversed"

### DISCUSSION THREAD

**Discussion by bn40400** [EricGoldsteinNz/tadpole#62] [General]
[BUG] When changing the background music through tadpole, tadpole replaces the new pagefile.sys file (the file that you want the new menu music to be) with the one that is already on the Datafrog, the one that you want to change. Basically in other words it appears to overwrite the new pagefile.sys file on the PC with the one that is already stored on the Datafrog (reversed). Simple workaround is to manually open the resources folder and replace the existing pagefile.sys with the new one.

**Comment by EricGoldsteinNz** [comment]
Thank you! Fixed in v1.2.1
